std-bits/table/src/main/java/org/enso/table/data/column/operation/map/MapOpStorage.java

```diff
@@ -24,7 +24,7 @@ public class MapOpStorage<T, S extends Storage<? super T>> {
    * @return whether the operation is supported
    */
   public boolean isSupported(String n) {
-    return ops.get(n) != null;
+    return n != null && ops.get(n) != null;
```

Comment:
When can `n` be `null`?

Reply:
Called from Enso and was useful for an experiment to put it here rather than the top level.